internal/attacktechniques/k8s/privilege-escalation/hostpath-volume/main.tf

```diff
@@ -0,0 +1,37 @@
+terraform {
+  required_providers {
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = "2.7.1"
+    }
+  }
+}
+
+variable "kubeconfig" {
```

Comment:
It's probably not useful to add it as a variable, since we don't provide any interface to change it. I'll switch to using a `local`